code/modules/mob/living/carbon/human/human.dm

```diff
@@ -936,6 +937,25 @@
 			var/msg = span_notice("[key_name_admin(usr)] has removed [key_name(src)] from purrbation.")
 			message_admins(msg)
 			admin_ticket_log(src, msg)
+	//SKYRAT EDIT ADDITION BEGIN - CRYO SEND
+	if(href_list[VV_HK_SEND_CRYO])
+		if(!check_rights(R_SPAWN))
+			return
+		var/choisen = tgui_alert(usr, "Add a paper notice about sending [name] into a cryopod?", "Leave a paper?", list("Yes", "No", "Cancel"))
+		if(choisen == "Cancel" || !choisen)
+			return
+		else
+			//log/message
+			to_chat(usr, "Put [src] in cryopod.")
+			log_admin("[key_name(usr)] has put [key_name(src)] into a cryopod.")
+			var/msg = span_notice("[key_name_admin(usr)] has put [key_name(src)] into a cryopod from [ADMIN_VERBOSEJMP(src)].")
+			message_admins(msg)
+			admin_ticket_log(src, msg)
+			
+			choisen = choisen == "Yes"
+		send_to_cryo(choisen)
```

Comment:
Since you did an early return, you don't need the `else` statement, the goal of doing those is to help reduce indentation.
```suggestion

		//log/message
		to_chat(usr, "Put [src] in cryopod.")
		log_admin("[key_name(usr)] has put [key_name(src)] into a cryopod.")
		var/msg = span_notice("[key_name_admin(usr)] has put [key_name(src)] into a cryopod from [ADMIN_VERBOSEJMP(src)].")
		message_admins(msg)
		admin_ticket_log(src, msg)

		send_to_cryo(send_notice == "Yes")
```